constants/index.ts

```diff
@@ -21,3 +21,5 @@ export const PROTECTED_ROUTES = [
 ];
 
 export const AUTH_ROUTES = [ROUTES.SIGN_UP, ROUTES.SIGN_IN];
+
+export const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL;
```

Comment:
As there will be multiple urls due to microservices so rename the url for the purpose it's related to